.expeditor/deploy.pipeline.yml

```diff
@@ -0,0 +1,24 @@
+steps:
+  - label: "Terraform"
+    command: .expeditor/buildkite/terraform.sh
+    concurrency: 1
```

Comment:
You need to specify a `concurrency_group` in both steps

Reply:
is there a default or do I need to create one somewhere?

Reply:
you just have to specify it like so: https://github.com/chef/a2/blob/master/.expeditor/deploy.pipeline.yml#L5